index.js

```diff
@@ -11,26 +11,50 @@
 let store = {};
 
 const _callsubscribers = (key, newValue, prevValue) => {
-  store[key].subscribers.forEach(callback => {
-    if (typeof callback === 'function') {
-      callback(newValue, prevValue);
+  store[key].subscribers.forEach(config => {
+    if (!config.options.onlyOnChange || !deepEqual(newValue, prevValue)) {
```

Comment:
This deepEqual call is gonna be made for every subscriber. It may be better if we call it just before the forEach, so we evaluate it only 1 time.
This will have even more sense if we set onlyOnChange in true by default

Reply:
Agreed! Will do it asap

Reply:
Delivered